baseplate/observers/tracing.py

```diff
@@ -179,6 +181,7 @@ def __init__(self, service_name, hostname, span, recorder):
         self.elapsed = None
         self.binary_annotations = []
         self.on_set_tag(ANNOTATIONS["COMPONENT"], "baseplate")
+
```

Comment:
style nit, we probably don't want to leave this extra blank line.

Reply:
thanks for catching!